force-app/main/default/lwc/accountModelSettings/accountModelSettings.html

```diff
@@ -1,123 +1,119 @@
 <template>
-    <div class="slds-grid--vertical slds-theme_default">
```

Comment:
all right, github cant even figure out which line is removed...

Anyway, we don't need these css classes since inheriting from parent component.
Also, we can move slds-p-top_large slds-p-left_small to edaSettingsPage.html so that each page component wont have to add this anymore.

Reply:
Have you tried rendering these on their own? We want them to be able to be inserted directly into modals.